webrtc/RTCIceCandidate-constructor.html

```diff
@@ -0,0 +1,178 @@
+<!doctype html>
+<title>RTCIceCandidate constructor</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+  'use strict';
+
+  const candidateString = 'candidate:1905690388 1 udp 2113937151 192.168.0.1 58041 typ host generation 0 ufrag thC8 network-cost 50'
+  const randomString = '<arbitrary string[0] content>;'
+
+  test(t => {
+    // The argument for RTCIceCandidateInit is optional (w3c/webrtc-pc#1153),
+    // but the constructor throws because both sdpMid and sdpMLineIndex are null
+    // Note that current browsers pass this test but may throw TypeError for
+    // different reason, i.e. they don't accept empty argument.
+    // Further tests below are used to differentiate the errors.
+    assert_throws(new TypeError,
+      () => new RTCIceCandidate());
+  }, 'new RTCIceCandidate()');
+
+  test(t => {
+    // All fields in RTCIceCandidateInit are optional,
+    // but the constructor throws because both sdpMid and sdpMLineIndex are null
+    // Note that current browsers pass this test but may throw TypeError for
+    // different reason, i.e. they don't allow undefined candidate string.
+    // Further tests below are used to differentiate the errors.
+    assert_throws(new TypeError,
```

Comment:
Agree, I personally prefer and use parenthesis for all new operators in my own code. I just saw the omittance of paren in some other test files and thought that's the style convention I should follow.

Anyway good to know people actually use both styles in this repo. I have updated the code to use the paren-all style we preferred.